google/cloud/aiplatform/training_jobs.py

```diff
@@ -1240,6 +1310,28 @@ def __init__(
             credentials (auth_credentials.Credentials):
                 Custom credentials to use to run call training service. Overrides
                 credentials set in aiplatform.init.
+            training_pipeline_encryption_spec_key_name (Optional[str]):
```

Comment:
I'd possibly remove `pipeline` from this name so it's consistent with the class name. If we end up changing this the XTrainingPipeline instead of XTrainignJob, we can add it back. You can leave the docstring as is.